src/DependencyInjection/SymfonycastsSassExtension.php

```diff
@@ -80,11 +86,39 @@ public function getConfigTreeBuilder(): TreeBuilder
                 ->scalarNode('binary')
                     ->info('The Sass binary to use')
                     ->defaultNull()
+                ->end()
+                ->arrayNode('sass_options')
+                    ->addDefaultsIfNotSet()
+                    ->children()
+                        ->booleanNode('charset')
+                            ->info('Whether to include the charset declaration in the generated Sass. (default: true)')
+                            ->defaultValue(true)
+                        ->end()
+                        ->enumNode('style')
+                            ->info('The style of the generated CSS: compressed or expanded (default: expanded)')
+                            ->values(['compressed', 'expanded'])
+                            ->defaultValue('expanded')
+                        ->end()
+                        ->booleanNode('source_map')
+                            ->info('Generate a source map file (default: true)')
+                            ->defaultValue(true)
+                        ->end()
+                        ->scalarNode('embed_source_map')
+                            ->info('Embed the sourcemap in the compiled CSS. By default, enabled only when debug mode is on.')
+                            ->defaultValue('%kernel.debug%')
+                        ->end()
+                        ->booleanNode('embed_sources')
+                            ->info('Embed the content of the Sass sources in the generated source map (default: false).')
+                            ->defaultValue(false)
+                        ->end()
                     ->end()
+                    ->ignoreExtraKeys()
```

Comment:
-1 for that. Ignoring extra keys means that any mistake in the config will be silently ignored which leads to much worse DX.